.github/workflows/deploy.pr.yml

```diff
@@ -0,0 +1,63 @@
+name: Deploy PR
+
+# Run on pull request
+on: pull_request
+
+env:
+  REPO_NAME_ALPHANUMERIC: safereactapps
+
+jobs:
+  deploy:
+    name: Deploy Pull Request
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Remove broken apt repos [Ubuntu]
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          for apt_file in `grep -lr microsoft /etc/apt/sources.list.d/`; do sudo rm $apt_file; done
+
+      - uses: actions/checkout@v2
+
+      - name: Setup dependencies ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: |
+          sudo apt-get update
+          sudo apt-get -y install python3-pip python3-dev
+          pip install awscli --upgrade --user
+          # If you have dependencies from git (wc app), yarn may randomly throw an error about invalid cache
+          # This approach is taken from https://github.com/yarnpkg/yarn/issues/7212#issuecomment-506155894 to fix the issue
+          # Another approach is to install with flag --network-concurrency 1, but this will make the installation pretty slow (default value is 8)
+          mkdir .yarncache
+          yarn install --cache-folder ./.yarncache --frozen-lockfile
+          rm -rf .yarncache
+          yarn cache clean
+      - uses: actions/cache@v2
```

Comment:
I have one doubt here. Why are you calling cache action after `yarn build` ?

Reply:
It's yarn install, isn't it?

Reply:
Yes, I mean `yarn install`.

But I'm talking about the action used below. `actions/cache@v2`
Using it like that you are doing a full yarn install and then if there is a valid cache from previous build copying the files again

Reply:
But it caches node_modules, not the yarn cache?

Reply:
Oh, you mean that it should be before yarn install, don't you? I'm not sure. How will it cache files if they're not there yet? I think they are available inside the job every run by default, so you don't need to run it beforehand

Reply:
Ok, so I thought you found an special reason. ;)
That action first looks if there is something cached and try to load it. If node_modules has the expected dependencies versions (that it should because we are using yarn.lock in cache hash to check if it's compatible) `yarn install` would be way faster, so repeating builds will take less time

That action adds before Github clears everything up the "backup cache" step where it effectively stores the files to cache

Reply:
@dasanra please check again, thanks for catching it

Reply:
with the proper setup it shaved almost ~2m from the ci job, nice :)